docs/nlu/choosing-a-pipeline.rst

```diff
@@ -19,14 +19,20 @@ it on your dataset.
 The Short Answer
 ----------------
```

Comment:
add a note that this guidelines are approximate, and we recommend to experiment with different pipelines